components/alert/alert.js

```diff
@@ -100,28 +100,12 @@ class Alert extends LocalizeStaticMixin(RtlMixin(LitElement)) {
 		`];
 	}
 
-	static get resources() {
-		return {
-			'ar': { 'close': 'إغلاق التنبيه'},
-			'de': { 'close': 'Benachrichtigung schließen' },
-			'en': { 'close': 'Close Alert'},
-			'es': { 'close': 'Alerta de cierre' },
-			'fr': { 'close': 'Fermer l\'alerte' },
-			'ja': { 'close': 'アラートを閉じる' },
-			'ko': { 'close': '경보 닫기' },
-			'nl': { 'close': 'Waarschuwing sluiten' },
-			'pt': { 'close': 'Fechar Alerta' },
-			'sv': { 'close': 'Stängningsvarning' },
-			'tr': { 'close': 'Kapatma Uyarısı' },
-			'zh-tw': { 'close': '關閉警示' },
-			'zh': { 'close': '关闭警报' }
-		};
-	}
-
 	constructor() {
 		super();
 		this.hasCloseButton = false;
 		this.type = 'default';
+
+		this._namespace = 'components.alert.alert';
```

Comment:
Thoughts about flattening the namespace to just `component.<component-name>`? I think we'd be safe enough with `components.alert` and `components.alert-toast` and `components.input-date` for the namespace -- `components.alert.alert.close` seems like overkill.

Also, I think these namespaces could just be constant strings outside the class definition? No need to redefine it for each instance.

```javascript
const LANG_NAMESPACE = 'components.alert';
```